### PR TITLE
pingallHosts - allows users to specify hosts and pingtype to ping bet…

### PR DESCRIPTION
**Pull request by andrewonlab**
…ween all hosts to check for reachability

### CODE REVIEW

File: TestON/drivers/common/cli/emulator/mininetclidriver.py [hunk 1]
```diff
@@ -367,6 +367,64 @@ def fpingHost( self, **pingParams ):
...
+            Returns main.FALSE if one or more of hosts specified
+            cannot reach each other"""
+            
+        if pingType == "ipv4":
+            cmd = " ping -c 1 -i 1 -W 8 " 
+        elif pingType == "ipv6":
+            cmd = " ping6 -c 1 -i 1 -W 8 "
+        else:
+            main.log.warn( "Invalid pingType specified" )
+            return
```
Comment (jhall11) [inline TestON/drivers/common/cli/emulator/mininetclidriver.py]: You may want to add to the doc string that the function returns None on error

File: TestON/drivers/common/cli/emulator/mininetclidriver.py [hunk 2]
```diff
@@ -367,6 +367,64 @@ def fpingHost( self, **pingParams ):
...
+
+        try:
+            main.log.info( "Testing reachability between specified hosts" )
+           
+            isReachable = main.TRUE
+
+            for host in hostList:
```
Comment (hakrishn) [inline TestON/drivers/common/cli/emulator/mininetclidriver.py]: Check to see if using something like below reduces one extra for loop and make it more elegant.
hostCombos = list( itertools.combinations( hostList, 2 ) )
for i in xrange( 0, len( hostCombos )):
    pingCmd = hostCombos[i][0] + cmd + hostCombos[i][1]]

**Comment by kelvin-onlab** [comment]
I think this ready to merge?

**Comment by kelvin-onlab** [comment]
Accidentally closed the pull request. Sorry, Is this ready for merge?

**Comment by jhall11** [comment]
Yeah, I think Andrew can address any changes in another pull request since they aren't big changes

**Pull request merged**